components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -821,19 +821,29 @@ public String getOpenIDRealm() {
      * @return
      */
     public String getWstrustSP() {
-        if (wstrustEp != null) {
+        List<String> wsTrustEps = getAllWsTrustSPs();
+        if (CollectionUtils.isNotEmpty(wsTrustEps))
+            return getAllWsTrustSPs().get(0);
+        else
+            return null;
+    }
+
+    /**
+     * @return
+     */
+    public List<String> getAllWsTrustSPs() {
+        if (CollectionUtils.isNotEmpty(wstrustEp))
```

Comment:
enclose if, else statements in curly brackets.

Reply:
Addressed in my last commit. Thanks.